src/Umbraco.Web/UmbracoApplicationBase.cs

```diff
@@ -15,7 +14,14 @@ namespace Umbraco.Web
     /// </summary>
     public abstract class UmbracoApplicationBase : HttpApplication
     {
-        private IRuntime _runtime;
+        // needs to be static, if and exception is thrown in Application_Start the current HttpApplication is discarded
+        // and a new instance is created, meaning we loose value if it's already been set.
+        private static IRuntime _runtime;
+
+        /// <summary>
+        /// Gets the current <see cref="IRuntime"/>.
+        /// </summary>
+        protected IRuntime Runtime => _runtime;
```

Comment:
there's already a `protected  abstract IRuntime GetRuntime()` so don't think exposing this as protected should be done?

Reply:
`GetRuntime()` creates a new instance when called, but given I didn't use it anyway I've removed the property